go.mod

```diff
@@ -1,32 +1,58 @@
 module timestream-prometheus-connector
 
-go 1.14
+go 1.22.3
 
 require (
-	github.com/Microsoft/go-winio v0.6.1 // indirect
 	github.com/alecthomas/kingpin/v2 v2.4.0
 	github.com/aws/aws-lambda-go v1.46.0
-	github.com/aws/aws-sdk-go v1.50.38
-	github.com/distribution/reference v0.5.0 // indirect
-	github.com/docker/distribution v2.8.3+incompatible // indirect
+	github.com/aws/aws-sdk-go v1.52.4
```

Comment:
This is the latest version but still has a vulnerability according to `osv-scanner`
```
╭──────────────────────────────┬──────┬───────────┬───────────────────────────┬─────────┬────────╮
│ OSV URL                      │ CVSS │ ECOSYSTEM │ PACKAGE                   │ VERSION │ SOURCE │
├──────────────────────────────┼──────┼───────────┼───────────────────────────┼─────────┼────────┤
│ Uncalled vulnerabilities     │      │           │                           │         │        │
├──────────────────────────────┼──────┼───────────┼───────────────────────────┼─────────┼────────┤
│ https://osv.dev/GO-2022-0646 │      │ Go        │ github.com/aws/aws-sdk-go │ 1.52.4  │ go.mod │
╰──────────────────────────────┴──────┴───────────┴───────────────────────────┴─────────┴────────╯
```

Reply:
This is confirmed as a known issue. We fortunately don't call any of the vulnerable methods.